app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
We do not use this controller anywhere, so we do not need to commit this controller or any commented code